packages/machine/test/unit/models/state-channel/set-state.spec.ts

```diff
@@ -47,7 +47,7 @@ describe("StateChannel::setState", () => {
         token: AddressZero
       },
       false,
-      Math.ceil(Math.random() * 2e10),
+      2,
```

Comment:
Shouldn't this be 1?

ETHFreeBalance = 0
TestApp = 1

?

Reply:
That's what I thought as well. It fails with `1` though, and I didn't spend the time investigating why.

Reply:
Ok, I'll look into it